### ISSUE TITLE
Import user avatars

### ISSUE THREAD

**Issue by lichtmetzger** [lichtmetzger/iwar-theme#1]
Avatars are not imported by default, because bbPress doesn't support them.

I need to write some custom code for the FG Joomla importer to integrate with this plugin:
https://de.wordpress.org/plugins/basic-user-avatars/

(Or pay the developer for that feature, but what's the fun in that?)

**Comment by lichtmetzger** [comment]
Switching to [Simple Local Avatars](https://wordpress.org/plugins/simple-local-avatars/), because it seems to be maintained a lot more regularly and has a bigger install base.

**Comment by lichtmetzger** [comment]
The avatar is saved in the usermeta table, based on the user_id key.

basic_user_avatar: JSON object with multiple image sizes

**Comment by lichtmetzger** [comment]
This is how it's saved by the plugin, we need to migrate it over to the importer plugin:

```
$avatar = wp_handle_upload( $_FILES['basic-user-avatar'], array( 'mimes' => $mimes, 'test_form' => false, 'unique_filename_callback' => array( $this, 'unique_filename_callback' ) ) );

// Save user information (overwriting previous)
update_user_meta( $user_id, 'basic_user_avatar', array( 'full' => $avatar['url'] ) );
```

That meta value also contains autogenerated thumbnail sizes, but it seems like we don't have to generate them on import, since there's a check in get_avatar_data() that should do this for us later.

**Comment by lichtmetzger** [comment]
Implemented with ccccbe16aea57ca95069457eae6e7ad9c6d70cc7.
